postgresql/shell-scripts/unix/drop-db-changelog.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
```

Comment:
Also rename to "drop-changelog" (see comment above)

Reply:
updated with [f85d574](https://github.com/3dcitydb/3dcitydb/pull/226/commits/f85d57478a4343404e944d8b1d2a5231ff47bfe4)